piet-coregraphics/src/ct_helpers.rs

```diff
@@ -384,6 +384,23 @@ impl FontCollection {
     }
 }
 
+// the version of this in the coretext crate doesn't let you supply an affine
+#[allow(clippy::many_single_char_names)]
+pub(crate) fn make_font(desc: &CTFontDescriptor, pt_size: f64, affine: Affine) -> CTFont {
+    let [a, b, c, d, e, f] = affine.as_coeffs();
+    let affine = CGAffineTransform::new(a, b, c, d, e, f);
+    unsafe {
+        let font_ref = CTFontCreateWithFontDescriptor(
+            desc.as_concrete_TypeRef(),
+            pt_size as CGFloat,
+            // ownership here isn't documented, which I presume to mean it is copied,
+            // and we can pass a stack pointer.
+            &affine as *const _,
```

Comment:
You don't need the `as` here, it should automatically convert the reference into the appropriate pointer.